beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockFactoryDenebTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.coordinator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.TestSpecFactory;
+import tech.pegasys.teku.spec.datastructures.blobs.versions.deneb.BlobsBundle;
+import tech.pegasys.teku.spec.datastructures.blocks.BlindedBlockContainer;
+import tech.pegasys.teku.spec.datastructures.blocks.BlockContainer;
+import tech.pegasys.teku.spec.datastructures.blocks.versions.deneb.BlindedBlockContents;
+import tech.pegasys.teku.spec.datastructures.blocks.versions.deneb.BlockContents;
+import tech.pegasys.teku.spec.util.DataStructureUtil;
+
+public class BlockFactoryDenebTest extends AbstractBlockFactoryTest {
+
+  private final Spec spec = TestSpecFactory.createMinimalDeneb();
+
+  @Test
+  void shouldCreateBlockContentsWhenDenebIsActive() {
+
+    prepareDefaultPayload(spec);
+    final BlobsBundle blobsBundle = prepareBlobsBundle(spec);
+
+    final BlockContainer blockContainer = assertBlockCreated(1, spec, false, false);
+
+    assertThat(blockContainer).isInstanceOf(BlockContents.class);
+    assertThat(blockContainer.getBlobSidecars())
```

Comment:
I'd clarify before this assertion there are more than 0 blobSidecars in these and all other tests. If by some error there are 0 blobs there, we are not testing anything here

Reply:
good point, I made blobs bundle fixed size.